Gemfile.lock

```diff
@@ -1,223 +1,13 @@
-      terminal-notifier (>= 2.0.0, < 3.0.0)
-      terminal-table (>= 1.4.5, < 2.0.0)
-      tty-screen (>= 0.6.3, < 1.0.0)
-      tty-spinner (>= 0.8.0, < 1.0.0)
-      word_wrap (~> 1.0.0)
-      xcodeproj (>= 1.13.0, < 2.0.0)
-      xcpretty (~> 0.3.0)
-      xcpretty-travis-formatter (>= 0.0.3)
-    fourflusher (2.3.1)
-    fuzzy_match (2.0.4)
-    gh_inspector (1.1.3)
-    google-api-client (0.36.4)
-      addressable (~> 2.5, >= 2.5.1)
-      googleauth (~> 0.9)
-      httpclient (>= 2.8.1, < 3.0)
-      mini_mime (~> 1.0)
-      representable (~> 3.0)
-      retriable (>= 2.0, < 4.0)
-      signet (~> 0.12)
-    google-cloud-core (1.5.0)
-      google-cloud-env (~> 1.0)
-      google-cloud-errors (~> 1.0)
-    google-cloud-env (1.3.0)
-      faraday (~> 0.11)
-    google-cloud-errors (1.0.0)
-    google-cloud-storage (1.25.1)
-      addressable (~> 2.5)
-      digest-crc (~> 0.4)
-      google-api-client (~> 0.33)
-      google-cloud-core (~> 1.2)
-      googleauth (~> 0.9)
-      mini_mime (~> 1.0)
-    googleauth (0.10.0)
-      faraday (~> 0.12)
-      jwt (>= 1.4, < 3.0)
-      memoist (~> 0.16)
-      multi_json (~> 1.11)
-      os (>= 0.9, < 2.0)
-      signet (~> 0.12)
-    highline (1.7.10)
-    http-cookie (1.0.3)
-      domain_name (~> 0.5)
-    httpclient (2.8.3)
-    i18n (0.9.5)
-      concurrent-ruby (~> 1.0)
-    json (2.3.0)
-    jwt (2.1.0)
-    memoist (0.16.2)
-    mini_magick (4.10.1)
-    mini_mime (1.0.2)
-    minitest (5.14.0)
-    molinillo (0.6.6)
-    multi_json (1.14.1)
-    multi_xml (0.6.0)
-    multipart-post (2.0.0)
-    nanaimo (0.2.6)
-    nap (1.1.0)
-    naturally (2.2.0)
-    netrc (0.11.0)
-    os (1.0.1)
-    plist (3.5.0)
-    public_suffix (2.0.5)
-    representable (3.0.4)
-      declarative (< 0.1.0)
-      declarative-option (< 0.2.0)
-      uber (< 0.2.0)
-    retriable (3.1.2)
-    rouge (2.0.7)
-    ruby-macho (1.4.0)
-    rubyzip (1.3.0)
-    security (0.1.3)
-    signet (0.12.0)
-      addressable (~> 2.3)
-      faraday (~> 0.9)
-      jwt (>= 1.5, < 3.0)
-      multi_json (~> 1.10)
-    simctl (1.6.7)
-      CFPropertyList
-      naturally
-    slack-notifier (2.3.2)
-    terminal-notifier (2.0.0)
-    terminal-table (1.8.0)
-      unicode-display_width (~> 1.1, >= 1.1.1)
-    thread_safe (0.3.6)
-    tty-cursor (0.7.1)
-    tty-screen (0.7.0)
-    tty-spinner (0.9.2)
-      tty-cursor (~> 0.7)
-    tzinfo (1.2.6)
-      thread_safe (~> 0.1)
-    uber (0.1.0)
-    unf (0.1.4)
-      unf_ext
-    unf_ext (0.0.7.6)
-    unicode-display_width (1.6.1)
-    word_wrap (1.0.0)
-    xcodeproj (1.14.0)
-      CFPropertyList (>= 2.3.3, < 4.0)
-      atomos (~> 0.1.3)
-      claide (>= 1.0.2, < 2.0)
-      colored2 (~> 3.1)
-      nanaimo (~> 0.2.6)
-    xcpretty (0.3.0)
-      rouge (~> 2.0.7)
-    xcpretty-json-formatter (0.1.1)
-      xcpretty (~> 0.2, >= 0.0.7)
-    xcpretty-travis-formatter (1.0.0)
-      xcpretty (~> 0.2, >= 0.0.7)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
-  cocoapods
```

Comment:
@AvdLee Do you know why we used to have the gems fastlane, pretty etc in here but now only the dotenv gem?

Reply:
I would not be surprised if that's a result of copying files from other projects or using a general setup for the project. They can be safely removed at least!